src/space/objects/pattern/collisionHandler.js

```diff
@@ -54,11 +59,16 @@ class EncounterCollisionHandler {
         loopHole.transfer(object);
     }
 
+    handleBulletCollision(bullet, object) {
+        bullet.hit(bullet.health);
+        object.hit(BULLET_DAMAGE);
```

Comment:
I think bullets should get destroyed upon collision with another game object. This way I got hit one and even tho bullet damage is set to 8, my ship was destroyed in few frames.

Reply:
I thought that the bullet gets destroyed in hit method when the bullet health reaches zero